### ISSUE TITLE
Tidy up tabs and add links

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#865]
We want to give users a richer experience with more information / pointers for each species once they drill down to that level

Note issues #863 and #864 

At the same time, the tabs appearing on each leaf are expanding and potentially getting messy.  

The IUCN Red List tab already only contains a link because the page we wanted in the tab doesn't allow iframing.

I would propose that we tidy up the tabs interface and include only the most essential ones as tabs whilst at the same time have a new tab which contains a collection of external links.
 
An alternative paradigm that could be adopted would be to generate a single scrollable page that we generate and which has sections of information about the species derived from our database / wikipedia page / GBIF / sponsorship and then had links underneath.

**Comment by hyanwong** [comment]
For IUCN we could use their API to construct a page, like we do for wikipedia.

**Comment by hyanwong** [comment]
I think the tab interface is more appealing, and doesn't get us into trouble re combining our data with that from other sources. E.g. IUCN will probably be OK with having all their data in one tab, but may not like it if it appears that we are "augmenting" one of our own pages with their data.

**Comment by lentinj** [comment]
I'm not sure there's much middle ground IMO, we either have tabs as we do today or we switch to a summary page with links as was floating about with the Tours UI redesign. We could have more tabs in the expert mode I guess, but I don't think they'll be discoverable enough to be worthwhile.

I think I prefer the idea of a summary page with links meself, even if it does run the risk of basically re-implementing the wikipedia pages with OneZoom theming, I think keeping the site in a consistent OneZoom theme is still important. And it'll be a lot simpler to implement than the current tabs. However, it's certainly not the case that no-tabs is categorically better than tabs.

> IUCN will probably be OK with having all their data in one tab, but may not like it if it appears that we are "augmenting" one of our own pages with their data.

I suspect I've asked this before, but [wikipedia infoboxes seem to have IUCN embedded](https://en.wikipedia.org/wiki/Little_penguin) with remarkably little branding at that. Is this just because they're Wikipedia?

**Comment by hyanwong** [comment]
It's not really the IUCN status, it's more if we use their API to get textual summaries, I think.

My guess is it would be hard to have both wikipedia (no links) and OneZoom, etc. I do still prefer tabs, FWIW.

**Comment by davidebbo** [comment]
FWIW, I've never been a fan of the embedded wikipedia page. I almost always end up opening in a new tab. I guess it just looks cleaner and makes use of the full screen size. Maybe it's just me!

BTW, the link at the bottom states "This is a summary", but it seems to me it's the full page that's embedded, no?

**Comment by hyanwong** [comment]
The embedded page allows us to remove links etc if need be (e.g. for museums). It doesn't have the surrounding wikipedia titlebars / sidebars but I think is otherwise identical. Personally, I like it, but I can see why you might want to go straight to wikipedia instead.

**Comment by davidebbo** [comment]
> The embedded page allows us to remove links etc if need be (e.g. for museums)

Of, so we're prefetching/storing all the wikipedia pages for museums? That's a ton of data!

**Comment by hyanwong** [comment]
No, there's a REST API which funnels the data from wikipedia straight to the client (I seem to remember: I don;t think it comes via us first, but it might just relay it on)

**Comment by lentinj** [comment]
> I don't think it comes via us first, but it might do and we just relay it on after removing links

We don't proxy the wikipedia calls, this all happens clientside. Your browser fetches the page via the Wikipedia API, optionally filters links and pops the result in the iframe.

> Of, so we're prefetching/storing all the wikipedia pages for museums? That's a ton of data!

It's not something OneZoom uses, but it'd be easy to arrange with Kiwix: https://wiki.kiwix.org/wiki/Content_in_all_languages

**Comment by lentinj** [comment]
Here's my suggestion to unpick this:

1. A new summary page that has:
  * The first paragraph from the Wikipedia page (like your favourite search engine)
  * The image for that species, along with the copyright information
  * Sponsorship information, current sponsor name and story or the QR-code / link to sponsor (which then opens a new tab)
  * Available tours from that leaf/node
  * Any potentially embeddable content, à la https://www.gbif.org/species/2477459
2. Include the summary page...
  * ...as the first & default tab for onezoom.org, supplanting Wikipedia
  * ...as the only tab for Museum displays, so there's no way to leave the onezoom sandbox
4. Remove the now redundant:
  * Image copyright popup
  * Sponsorship tab
  * Tours tab
  * Image / sponsorship active areas (A leaf/node should be only one link IMO)
  * The link-less version of Wikipedia (We have 2 options, only the summary page, or tabs with full-fat versions of pages)

The final point is possibly over-reaching, but I think it's worth considering. The link-less Wikipedia pages bug me a bit, both editorially (the content is going to be expecting links that aren't there) and technically (we're hosting Wikipedia HTML under a onezoom.org origin, which isn't ideal). If it's something museums specifically request then fair enough, but I'm guessing the tours are more interesting nowadays.

How we get that first paragraph of Wikipedia content I'm not sure, I don't know the available APIs well enough atop my head. Depends if we want to offer a truly offline version or not really.

**Comment by jrosindell** [comment]
Current idea is to have a summary page which opens by default and replaces the sponsorship tab.  It should include a range of basic overview data.

The tabs should be scrollable to allow more

At the Bottom of the summary page there should be a further reading section which distinguishes between tabs and links

In the middle of the page, tabs which associate with content can be flagged where they appear e.g. read more on wikipedia.

**Comment by jrosindell** [comment]
We may also have the ability to configure the prominent tabs

**Comment by lentinj** [comment]
Things we mentioned that could go into the summary page:

* Title: Name / Scientific name
* Wikipedia summary, pulled from API, below an "open wikipedia tab" link
* Navigation (i.e. the content of the location dropdown), linking back to places in the tree
* IUCN widget, linking directly to IUCN site (which is all the IUCN tab does anyway)
* Species popularity
* Right hand image, including copyright owner & link below (superseding image information pop-up)
* Further reading links, with all known sources (including things that might be in tabs, and other stuff in the ozlinks tab).

A museum display wouldn't include any other tabs, and the tab bar would then hide as a result.

**Comment by jrosindell** [comment]
Note a comment from a user "From a leaf there is no easy way to directly copy the common name or the species binomial. You have to click to the wiki page and copy from there then format-less paste as its usually italic + bold on wiki. I don't know how easy it is to make the text element in the nodes copiable even via right click would be useful." which suggests that a summary page from us would resolve, especially if we also try to channel users into clicking on the summary page more clearly when they have zoomed all the way into a leaf - maybe clicking on a leaf already zooms in and opens the summary page automatically as split screen or shows a much more obvious button for doing so.
